### ISSUE TITLE
vvv now runs on Virtualbox 5.2? 

### ISSUE THREAD

**Issue by skorasaurus** [Varying-Vagrant-Vagrants/VVV#1402]
Hi, 

The TLDR: 
vvv works on virtualbox 5.2.4; possibly can removed the requirement from the docs (originally added at https://github.com/Varying-Vagrant-Vagrants/varyingvagrantvagrants.org/pull/48) 

Before I make a PR, I'd want someone else to confirm their success of running vvv on virtualbox 5.2 while on ubuntu...

=============================================

A bit of context: 
I updated my kernel on ubuntu to 4.13.0-26-generic late yesterday to deal w/ the meltdown/spectre vulnerabilities; 

I went to my vvv, vagrant up and got an error: 

`vagrant failed to initialize at a very early stage:

The provider 'virtualbox' that was requested to back the machine
'default' is reporting that it isn't usable on this system. The
reason is shown below:

VirtualBox is complaining that the kernel module is not loaded. Please
run `VBoxManage --version` or open the VirtualBox GUI to see the error
message which should contain instructions on how to fix this error.
myusername@mybox:~/vvv$ VBoxManage --version
```
WARNING: The vboxdrv kernel module is not loaded. Either there is no module
         available for the current kernel (4.13.0-26-generic) or it failed to
         load. Please recompile the kernel module and install it by

           sudo /sbin/vboxconfig

         You will not be able to start VMs until this problem is fixed.
5.1.22r115126 (current version of my VM)`
```

So, I thought: 
my virtualbox (5.1.22r115126) is likely outdated and I should update my virtualbox to the newest public release
or I could go back and use my previous (vulnerable) kernel or 
try to get the right kernel module to load on virtualbox 5.1 (and I have no idea how to do this), so I rashly chose option 1. 

I downloaded and installed the newest version of virtualbox, for me it was [5.2_5.2.4-119785](http://download.virtualbox.org/virtualbox/5.2.4/virtualbox-5.2_5.2.4-119785~Ubuntu~xenial_amd64.deb); 
I installed it ```sudo dpkg -i thatfilename.deb ; ran vagrant up in my vvv ran into another error - that my version of vagrant (1.9.1) didn't support virtualbox 5.2.....
so I downloaded Vagrant 2.0.1 and installed it.....  went back to my vvv folder, ran vagrant up, 

and got: 
```
Vagrant failed to initialize at a very early stage:

The plugins failed to initialize correctly.
```
I thought to update the plugins; I did that (vagrant plugin update); which updated a couple plugins 
(Updated 'vagrant-share' to version '1.1.9'!
Updated 'vagrant-vbguest' to version '0.15.1'!
for reference) and ran 
`vagrant plugin install vagrant-vbguest vagrant-hostsupdater` for good measure... which reported back

```
Installed the plugin 'vagrant-vbguest (0.15.1)'!
Installing the 'vagrant-hostsupdater' plugin. This can take a few minutes...
Installed the plugin 'vagrant-hostsupdater (1.0.2)'!
```

and went to go vagrant up in my vvv one more time....

and it ran some updates in there; and VVV works for me.

=======================================

So, the reason I'm writing this really long yak-shaving is, because there might be other vvv users on ubuntu running into this problem very soon (if they haven't already) and wanted to update. 

I don't know why vbox 5.1 is recommended - perhaps @BandonRandon can share why 5.2 wasn't working for vvv](https://github.com/Varying-Vagrant-Vagrants/varyingvagrantvagrants.org/pull/48) (upon a cursory google - perhaps it's from this https://github.com/hashicorp/vagrant/issues/9090 - which has been resolved.?)

**Comment by tomjn** [comment]
There are some versions of 5.2 that have issues with Vagrant 2.0. Which versions I do not know.

It's all a moot point though as VirtualBox 5.3.x is available, there's no reason to use 5.2.x or 5.1.x

**Comment by tomjn** [comment]
So nothing in particular has been done by VVV, the problem was enver that VVV didn't work, but that there were issues deeper down the pipeline in the Vagrant/Virtualbox parts

Whatever the issue was, it's probably fixed by a newer version of the Virtualbox guest editions, or an updated box
